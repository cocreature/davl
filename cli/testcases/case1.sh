#!/usr/bin/env bash
set -euo pipefail

cd "$(dirname ${BASH_SOURCE[0]})"

CLI=$(stack exec which davl-cli)

echo '!Bob' | $CLI
echo | $CLI Bob
echo '!Alice' | $CLI Bob
echo | $CLI
