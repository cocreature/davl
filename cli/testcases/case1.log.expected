[94mreplaying 0 transactions[97m[96mtype "help" to see available commands[97m
[92m'Alice'> [97m[92m'Alice'> [97m[94mreplaying 1 transactions[97m[96mtype "help" to see available commands[97m
[92m'Bob'> [97m[96mGift [#0:0] <-- 'Alice'
[97m
[92m'Bob'> [97m[94mreplaying 1 transactions[97m[96mtype "help" to see available commands[97m
[92m'Bob'> [97m[92m'Bob'> [97m[94mreplaying 2 transactions[97m[96mtype "help" to see available commands[97m
[92m'Alice'> [97m[96mGift [#0:0] --> 'Bob'
Gift [#1:0] <-- 'Bob'
[97m
[92m'Alice'> [97m